### ISSUE TITLE
Continuing the discussion on office layout

### ISSUE THREAD

**Issue by benchristel** [software-patterns/hci-language#5]
About a week ago, @professor opened #2, presenting a counterpoint to my over-hasty claim that "open-plan offices don't work". See the [Neighborly Teams](https://github.com/software-patterns/workshop/blob/master/patterns/neighborly-teams.md) pattern for context. I merged the pull request, but I feel like it's worth digging deeper into this issue.

Since having a conversation on a closed PR is annoying, I'm opening this issue so we can have a place to discuss our experiences with various types of office plans.

I found [this Ph.D thesis](https://espace.library.uq.edu.au/view/UQ:401844) which investigates the situations in which open-plan offices are effective at promoting collaboration. It's long, but [this article](http://theconversation.com/open-plan-offices-can-actually-work-under-certain-conditions-89452) provides a summary.

@software-patterns/everyone here's a discussion question: thinking back on your career, what were your favorite spaces to work in? Describe them. We'll analyze them together. You can make ASCII-art floor plan doodles as I did in [Neighborly Teams](https://github.com/software-patterns/workshop/blob/master/patterns/neighborly-teams.md). Or, if you're a high-tech person with a camera, you can take pictures of real doodles.

**Comment by amilkh** [comment]
is quiet-office.md deleted?

**Comment by benchristel** [comment]
Good catch, @amilkh, I renamed it to [Neighborly Teams](https://github.com/software-patterns/workshop/blob/master/patterns/neighborly-teams.md) to reflect my changing thoughts about the matter.
